doc/user/content/sql/show-materialized-views.md

```diff
@@ -11,12 +11,14 @@ in Materialize.
 
 ## Syntax
 
-{{< diagram "show-materialized-views.svg" >}}
+```mzsql
+SHOW MATERIALIZED VIEWS [ FROM <schema_name> ] [ IN <cluster_name> ]
+```
 
-Field | Use
-------|-----
-_schema&lowbar;name_ | The schema to show materialized views from. Defaults to first resolvable schema in the search path. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
-_cluster&lowbar;name_ | The cluster to show materialized views from. If omitted, materialized views from all clusters are shown.
+Option                      | Description
+----------------------------|------------
+**FROM** <schema_name>      | The schema to show materialized views from. Defaults to first resolvable schema in the search path. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
```

Comment:
Per your suggestion from https://github.com/MaterializeInc/materialize/pull/30398#discussion_r1836191837 
-- Doing them in batches allowed me to do a sweep and I realized I might be letting the tail wag the dog in that what might turn out to be smaller exceptions shouldn't rule the majority. Even if decent number of exceptions, eh ... as long as we're consistent with those, we could just have 2 models.